### PR TITLE
docs: add @frsource/cypress-plugin-visual-regression-diff

### PR DESCRIPTION
**Pull request by FRSgit**
add @frsource/cypress-plugin-visual-regression-diff to the plugins page
new plugin conforms to the ["Adding Plugins" documentation](https://github.com/cypress-io/cypress-documentation/blob/master/CONTRIBUTING.md#adding-plugins)
this plugin works both with e2e & component-testing and with Cypress 10

### CODE REVIEW

**Comment by FRSgit** [comment]
Hey, can I do anything to push this one forward?

**Review by lmiller1990 — APPROVED** [review]

File: content/_data/plugins.json [hunk 1]
```diff
@@ -929,6 +929,20 @@
           ],
           "badge": "community"
         },
+        {
+          "name": "Visual Regression Diff",
+          "description": "Perform visual regression test with a nice GUI as help. 💅 Supports both e2e and component-testing",
```
Comment (lmiller1990) [inline content/_data/plugins.json:934]:
```suggestion
          "description": "Perform visual regression test with a nice GUI as help. 💅 Supports both E2E and Component Testing",
```

**Review by lmiller1990 — APPROVED** [review]
A little nicer with some capitalization maybe?

File: content/_data/plugins.json (hunk 1 above)
Comment (FRSgit) [reply content/_data/plugins.json:934]: Hey @lmiller1990! Sure, it is a bit nicer

Comment (FRSgit) [reply content/_data/plugins.json]: added: 3b178e95e6af4a8213e8b7c3ce4ba2220edd213e

**Comment by FRSgit** [comment]
Thanks!

**Comment by lmiller1990** [comment]
Thanks for the cool snapshot plugin :+1:

**Pull request merged**